skyagi/src/skyagi/context.py

```diff
@@ -19,14 +19,22 @@ def __init__(self, console: Console, openai_key: str, webcontext=None) -> None:
         self.observations_size_history = []
         self.webcontext = webcontext
 
-    def print(self, message: str, style: str = None):
+    def print(
+        self,
+        message: str,
+        role: str = "system",
+        msg_type: str = "info",
```

Comment:
nit: recommend creating enum types for role and msg_type to avoid unexpected typo.